ush/config.sh.RRFS_CONUS_3km

```diff
@@ -111,9 +112,8 @@ ENDHOUR="23"
 PREEXISTING_DIR_METHOD="upgrade" # "rename"
 INITIAL_CYCLEDEF="${DATE_FIRST_CYCL}0300 ${DATE_LAST_CYCL}2300 12:00:00"
 BOUNDARY_CYCLEDEF="${DATE_FIRST_CYCL}0000 ${DATE_LAST_CYCL}2300 06:00:00"
-PROD_CYCLEDEF="${DATE_FIRST_CYCL}0900 ${DATE_LAST_CYCL}2300 01:00:00"
-POSTPROC_CYCLEDEF="${DATE_FIRST_CYCL}0900 ${DATE_LAST_CYCL}2300 01:00:00"
-POSTPROC_LONG_CYCLEDEF="${DATE_FIRST_CYCL}0900 ${DATE_LAST_CYCL}2300 03:00:00"
+PROD_CYCLEDEF="00 01,02,04,05,07,08,10,11,13,14,16,17,19,20,22,23 ${CYCLEDAY} ${CYCLEMONTH} 2022 *"
```

Comment:
Is there any way to simplify this? It is a very long list.

Reply:
No way to simplify if we do long forecast every three hours.